src/libraries/System.Runtime/tests/TrimmingTests/InvariantGlobalizationTrue.cs

```diff
@@ -28,7 +28,9 @@ static int Main(string[] args)
         foreach (MemberInfo member in globalizationMode.GetMembers(allStatics))
         {
             // properties and their backing getter methods are OK
-            if (member is PropertyInfo || member.Name.StartsWith("get_"))
+            if ((member is PropertyInfo || member.Name.StartsWith("get_")) ||
+                (member is FieldInfo && member.Name.Contains("AllowInvariantCultureOnly")) ||
+                (member is ConstructorInfo && member.Name.Contains(".cctor")))
```

Comment:
This can be combined with the `IsWindows()` check below for `is ConstructorInfo`.

Also - no need to check the name, this code is only look at static members, and if the member is a ConstructorInfo, it must be a static constructor.